### PR TITLE
KAS1590 - Des/enhancement/styleguide bem structure

### PR DESCRIPTION
**Pull request by Wolfr**
This PR depends on the merging of #382 first.

* Improves the BEM structure of the styleguide.
* Removes nested components, this is unnecessary.
* Improves the BEM structure of the styleguide. BEM was used in an incorrect way.

Specific pointers for @brechtPhilips :

* As you can see in the specific commit https://github.com/kanselarij-vlaanderen/kaleidos-frontend/commit/3e76854c82310fcd52445b68b0bcee277a67b818 , I use  6 top level components instead of working in 1 "kitchensink.scss" file. This keeps styles of unrelated things out of each other's way.

This PR also slightly modifies the button documentation to work with the renewed structure. Specific button + documentation PR with Octane components is planned.

### CODE REVIEW

**Comment by woutersf** [comment]
Build onderweg:
http://kal-kastaar.s.redpencil.io:8080/job/kaleidos/job/FeatureBranches/job/DES%252Fenhancement%252Fstyleguide-bem-structure/

File: app/components/toolbar/group.js
Comment (rafael-mindreau) [inline app/components/toolbar/group.js]: You best merge the `DES/feature/panels` branch back into this one. Or wait until it's merged into `development` and merge that into this one - essentially it's the same, so that's up to you.

**Review by rafael-mindreau — COMMENTED** [review]
Awaiting merge of `DES/feature/panels`, the rest looks good. So essentially the approval is staged here! ✅

File: app/components/toolbar/group.js
Comment (Wolfr) [reply app/components/toolbar/group.js]: Yes, I'll do the branch juggling. But it seems most PRs are OK to just land into development. So I can merge them in order into development.

**Comment by MikiDi** [comment]
I understand correctly that this replaces the ember part for panels that is proposed in #382 ? Good to go for me.

**Comment by Wolfr** [comment]
@MikiDi This PR is only about the BEM structure of the styleguide.

**Comment by woutersf** [comment]
Ook hier de js fouten van 382.
Screenshot 2020-06-18 at 13 04 10
Ook ziet de layout in de agenda er wat raar uit.

**Comment by woutersf** [comment]
Ge-update met development en groene versie van #382 .

**Review by woutersf — APPROVED** [review]
✅ build ok
✅  code ok
Mergen maar!

**Pull request merged**
